recipes/config.rb

```diff
@@ -15,4 +15,7 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+# enable chef error handler
```

Comment:
You can avoid this comment because the action we're doing is already explicit in the name of the recipe we're including `enable_chef_error_handler` .